### ISSUE TITLE
Export to Obsidian on Android?

### ISSUE THREAD

**Issue by traptw1thin** [maxkrieger/voiceliner#107]
I'm not seeing a way to export the .md files to obsidian on android. Obsidian isn't one of the options that come up with the export option, and neither is my file system.

**Comment by katsumi27** [comment]
Yes, same issue for me. Honestly just being able to send it to my files / download locally would give me a good enough work around for me to just drop it in the right folder in my obsidian but that's not an option either.
